### PR TITLE
add ellipsis when there's more than one liner

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
Addresses https://github.com/dart-lang/dartdoc/issues/808

**Comment by sethladd** [comment]
cc @devoncarew @keertip

File: lib/src/model.dart [hunk 1]
```diff
@@ -62,13 +62,23 @@ abstract class Nameable {
...
+}
+
+abstract class ModelElement implements Comparable, Nameable, Documentable {
   final Element element;
   final Library library;
 
   ElementType _modelType;
   String _rawDocs;
-  Documentation _documentation;
+  Documentation __documentation;
```
Comment (devoncarew) [inline lib/src/model.dart]: the rare double underscore

File: lib/templates/_has_more_docs.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{{#hasMoreThanOneLineDocs}}
+<a href="{{href}}">&hellip;</a>
```
Comment (devoncarew) [inline lib/templates/_has_more_docs.html]: Interesting - did the … char not make it through the various file encodings?

You could also remove the `a href`. If just the line is long and gets truncated by the browser, it'll show an ellipses from the text-overflow css style. So, a plain ellipses char from a 2nd paragraph would match that.

**Comment by devoncarew** [comment]
lgtm! I like the solution.

File: lib/src/model.dart (hunk 1 above)
Comment (sethladd) [inline lib/src/model.dart]: Yeah, I'm not proud of that one. :(

File: lib/templates/_has_more_docs.html (hunk 2 above)
Comment (sethladd) [inline lib/templates/_has_more_docs.html]: > … char not make it through the various file encodings?

What is the ... char?

> You could also remove the a href.

And still show the hellip ?

**Pull request closed without merging**